web-app/src/pages/Profile/index.jsx

```diff
@@ -30,15 +32,15 @@ const Profile = props => {
           <div className="columns">
             <div>
               <h1>{totalHolidays - daysBooked}</h1>
-              <h4>Days Remaining</h4>
+              <h4><FontAwesomeIcon icon={faCalendar} /> Days Remaining</h4>
             </div>
             <div>
               <h1>{daysBooked}</h1>
-              <h4>Days Booked</h4>
+              <h4><FontAwesomeIcon icon={faCheck} /> Days Booked</h4>
```

Comment:
To keep this even more consistent with the team page append the word "Days" on to the end of the H1 number and remove it from the H4.